src/CMakeLists.txt

```diff
@@ -813,7 +815,7 @@ else()
 	endif()
 
 	if(MINGW)
-		set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -mwindows")
+		set(CMAKE_EXE_LINKER_FLAGS_RELEASE "-mwindows")
```

Comment:
I do not understand this one. What does this change? And why is this only needed for MINGW?

Reply:
lazily copied from docs:

>  This option is available for Cygwin and MinGW targets. It specifies that a GUI application is to be generated by instructing the linker to set the PE header subsystem type appropriately.

the `if(MSVC)` has it too, but with a different syntax.
it's also only needed at link time, if you use it for compiling files clang will print a warning.